docs/src/user_docs/assembly/flow_control.md

```diff
@@ -33,7 +33,7 @@ end
 where:
 
 * `instructions` can be a sequence of any instructions, including nested control structures.
-* `count` is the number of times the `instructions` sequence should be repeated (e.g. `repeat.10`). `count` must be an integer greater than $0$.
+* `count` is the number of times the `instructions` sequence should be repeated (e.g. `repeat.10`). `count` must be an integer or a constant greater than $0$.
```

Comment:
Let's make the word "constant" a link to the [constants](https://0xpolygonmiden.github.io/miden-vm/user_docs/assembly/code_organization.html#constants) section in the docs.